services/QuillLMS/app/services/classlink_integration/schools_fetcher.rb

```diff
@@ -3,15 +3,18 @@
 module ClasslinkIntegration
   class SchoolsFetcher < BaseFetcher
     RESOURCE = 'schools'
-    QUERY_STRING = "?filter=type='school'"
+    QUERY_STRING = "?filter=type='school'&fields=name,identifier"
 
     def api_response_selector_override = 'orgs'
 
     def transform_api_response(school_hash)
       {
         district_id:,
-        name: school_hash['name']
+        name: school_hash['name'],
+        nces_id: self.class.coerce_nces_id(school_hash['identifier'])
       }
     end
+
+    def self.coerce_nces_id(maybe_nces_id) = maybe_nces_id.to_s =~ /\A\d{12}\z/ ? maybe_nces_id : nil
```

Comment:
It's possible for Districts to use this field for non-NCES values, so validation is warranted